src/pages/LandingPages/MGCovid19/MGCovid19.vue

```diff
@@ -0,0 +1,285 @@
+						<p class="mg-headline" v-html="pageCopy.headline">
+						</p>
+						<p class="mg-subhead">
+							{{ pageCopy.subhead }}
+						</p>
+						<landing-form
+							:amount.sync="monthlyGoodAmount"
+							:selected-group.sync="selectedGroup"
+							:key="1"
+							:button-text="pageCopy.button"
+							v-if="!isMonthlyGoodSubscriber"
+						/>
+						<div class="already-subscribed-msg-wrapper" v-if="isMonthlyGoodSubscriber">
+							<h4>
+								You're already signed up for Monthly Good.
+								Changes to this contribution can be made in your
+								<a href="/settings/credit">credit settings</a>.
+							</h4>
+						</div>
+					</div>
+				</div>
+			</template>
+		</kv-hero>
+
+		<div class="benefits">
+			<m-g-covid-explained />
+		</div>
+
+		<div class="about-kiva row page-content">
+			<div class="columns">
+				<h2 class="impact-text text-center">
+					Over $1.4 billion in impact in 70+ countries
+				</h2>
+				<p>
+					Kiva is an international nonprofit, founded in 2005 in San Francisco, with a mission
+					to expand financial access to help underserved communities thrive and in some cases,
+					provide relief when crises may arise.
+				</p>
+				<p>
+					We do this by crowdfunding loans and unlocking capital for the underserved, improving
+					the quality and cost of financial services, and addressing the underlying barriers to
+					financial access around the world. Through Kiva's work, students can pay for tuition,
+					women can start businesses, farmers are able to invest in equipment and families can
+					afford needed emergency care.
+				</p>
+				<p class="featured-text text-center">
+					By lending as little as $25 on Kiva, you can be part of the solution and make a real
+					difference in someone’s life.
+				</p>
+				<h4 class="text-center">
+					100% of every dollar you lend on Kiva goes to funding loans.
+				</h4>
+			</div>
+		</div>
+
+		<div class="footnotes row page-content">
+			<div class="columns">
+				<h3>Footnotes:</h3>
+				<ul>
+					<li>
+						If you choose “One-time support”, the selected amount will only be deposited once.
+					</li>
+					<li>
+						If there is no longer a need for this fund, we’ll get in touch with you to ask
+						you to change your settings.
+					</li>
+					<!-- <li>
+						We will record prior category settings for previous Monthly Good subscribers.
+					</li> -->
+					<li>
+						Deposited funds for COVID-19 relief may be withdrawn or relent after they have
+						gone through 1 full loan cycle.
+					</li>
+				</ul>
+			</div>
+		</div>
+	</www-page>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import KvHero from '@/components/Kv/KvHero';
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+import MGCovidExplained from './MGCovidExplained';
+
+import LandingForm from './CovidLandingForm';
```

Comment:
This should probably be renamed to `covid-landing-form` to about confusion with the other component